Makefile

```diff
@@ -153,7 +152,6 @@ check: \
 	build \
 	check-format \
 	lint \
-	audit \
```

Comment:
I missed this when I committed the workaround - I only changed the drone file.

If you want to leave audit in with just that single warning ignored, you can change the audit target to do
```
$(CARGO) audit --ignore RUSTSEC-2020-0123
```

Reply:
k